tests/integration/test_spans.py

```diff
@@ -526,19 +526,19 @@ def test_span_ingestion(
                 "group": "8a97a9e43588e2bd",
                 "op": "resource.script",
             },
-            "span_id": "bd429c44b67a3eb1",
+            "span_id": "b0429c44b67a3eb1",
             "start_timestamp_ms": int(start.timestamp() * 1e3),
             "trace_id": "ff62a8b040f340bda5d830223def1d81",
         },
         {
             "description": r"test \" with \" escaped \" chars",
             "duration_ms": 1500,
             "exclusive_time_ms": 345.0,
-            "is_segment": True,
+            "is_segment": False,
```

Comment:
This span has a different segment_id than span_id, so it's not a segment.

Reply:
Can you put this as a comment in the code?